src/mmrelay/matrix_utils.py

```
@@ -345,22 +431,101 @@ def strip_quoted_lines(text: str) -> str:
     return " ".join(filtered).strip()
 
 
+async def handle_matrix_reply(room, event, reply_to_event_id, text, room_config, storage_enabled, local_meshnet_name):
```

Pylint (3.3.7)

[refactor] Too many arguments (7/5)

(R0913)